batch/batch/front_end/templates/batch.html

```diff
@@ -8,8 +8,8 @@ <h1>Batch {{ batch['id'] }}</h1>
   {% endfor %}
   {% endif %}
   <h2>Jobs</h2>
-    <div class="flex-col-align-right">
-      <div class="flex-col-align-right">
+  <div class="flex-col" style="width: 66%; min-width: 653px; max-width: 1024px;">
```

Comment:
66% of the viewport width to emulate the current styling, I think 100% looks cleaner